scripts/models/qwen3-4B-2507.sh

```diff
@@ -0,0 +1,17 @@
+MODEL_ARGS=(
+   --swiglu
+   --num-layers 36
+   --hidden-size 2560
+   --ffn-hidden-size 9728
+   --num-attention-heads 32
+   --group-query-attention
+   --num-query-groups 8
+   --use-rotary-position-embeddings
+   --disable-bias-linear
+   --normalization "RMSNorm"
+   --norm-epsilon 1e-6
+   --rotary-base 5000000
+   --vocab-size 151936
+   --kv-channels 128
+   --qk-layernorm
+)
```

Comment:
we don't really need this. you can still source the origin 4B config and set `--rotary-base 5000000`